rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date, book, person)
+    @date = date
+    @person = person
+    @book = book
+
+    # associations
+    book.rentals << self
+    person.rental << self
```

Comment:
- [ ] Based on what is defined in your Person class this should be `person.rentals`, not `person.rental`. Kindly fix that.

Reply:
Fixed that statement.